src/Controllers/BatchProcessorController.ts

```diff
@@ -415,19 +431,59 @@ module StreamStats.Controllers {
                     }
 
                     else {
-                        // this.streamStatsAvailable = false;
-                        // this.toaster.pop('error', "No parameters available for this region", "Please select another region");
                     }
                     return paramRaw;
                 }, (error) => {
-                    // console.log('Bad response from the regression service');
-                    // this.streamStatsAvailable = false;
-                    // this.toaster.pop('warning', "StreamStats not available here at this time", "", 5000);
-
                 }).finally(() => {
                 });
         }
 
+        public submitBatch(): void {
+
+            // validate that the file is a .zip
+            if (this.submitBatchData.attachment.type != "application/zip") {

```

Comment:
When I try submitted my zip file I am still seeing this error pop up. My attachment.type is `application/x-zip-compressed`, maybe we have to check for both types.

![image](https://user-images.githubusercontent.com/59063181/231861360-ae711493-8670-4749-8383-86f14d9cdc79.png)